labs/lab4-soluce.md

```diff
@@ -1,3 +1,259 @@
 # Soluces for Lab 4
 
-TODO
+## Prerequisite:
```

Comment:
Could you move this part to step0 so people can do that prior to the lab or while we are doing the introduction?

Reply:
DONE

Reply:
I don't see any modification in `step0.md`. Could you check?

Reply:
@alsayasneh Could you move the Prerequisite part to `step0.md` file?